.editorconfig

```diff
@@ -244,6 +244,9 @@ dotnet_diagnostic.CA1846.severity = suggestion
 dotnet_diagnostic.CA2008.severity = suggestion
 # CA2012: Use ValueTask correctly
 dotnet_diagnostic.CA2012.severity = suggestion
+# IDE0044: Make field readonly
+dotnet_diagnostic.IDE0044.severity = suggestion
```

Comment:
Is this going to create more errors in other slnf's that only show up in VS?

Reply:
This is just changing the error to a suggestion for test/sample projects. The option is currently applying to our src projects and causing IDE only errors (which is annoying, why are there IDE only errors?!?)